packages/frontend/lib/content.d.ts

```diff
@@ -14,8 +14,10 @@ interface RichLinkBlockElement {
 interface ImageBlockElement {
     _type: 'model.dotcomrendering.pageElements.ImageBlockElement';
     media: { allImages: Image[] };
-    data: { alt: string; credit: string; caption?: string };
+    data: { alt: string; credit: string; caption?: string; copyright?: string; };
```

Comment:
Just checking, but is this needed for non-main images too?

Reply:
I have seen it on non-main images ([example](https://www.theguardian.com/games/2019/jan/04/rainbow-arcade-lgbtq-video-games-exhibition-world-of-warcraft-nintendo-berlin.json?guui))